### PR TITLE
Remove unused artifacts for common and parent module

### PR DESCRIPTION
**Pull request by yaooqinn**
### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -532,6 +532,10 @@
                 <artifactId>libthrift</artifactId>
                 <version>${thrift.version}</version>
                 <exclusions>
+                    <exclusion>
+                        <groupId>org.apache.commons</groupId>
+                        <artifactId>commons-lang3</artifactId>
```
[the remark this answers is not in the document]
Comment (pan3793) [reply pom.xml:537]: Then please also exclude apache http clients jars

[the remark this answers is not in the document]
Comment (yaooqinn) [reply pom.xml:537]: nice suggestion

**Review by pan3793 — APPROVED** [review]

File: kyuubi-metrics/pom.xml [hunk 2]
```diff
@@ -88,6 +88,11 @@
             <artifactId>jackson-databind</artifactId>
         </dependency>
 
+        <dependency>
```
Comment (SteNicholas) [inline kyuubi-metrics/pom.xml:91]: Does the guava package need to be shaded?

**Review by SteNicholas — COMMENTED** [review]
@yaooqinn, thanks for your contribution. I left the minor comments for the guava package which need to be shaded.

File: kyuubi-metrics/pom.xml (hunk 2 above)
Comment (ulysses-you) [reply kyuubi-metrics/pom.xml:91]: it is only used at Kyuubi Server, so there is no other version of guava

**Review by ulysses-you — APPROVED** [review]

File: kyuubi-metrics/pom.xml (hunk 2 above)
Comment (SteNicholas) [reply kyuubi-metrics/pom.xml:91]: > it is only used at Kyuubi Server, so there is no other version of guava

@ulysses-you, no matter whether the guava package is only used at Kyuubi Server, the guava package is better to be shaded for the users.

Comment (pan3793) [reply kyuubi-metrics/pom.xml:91]: I'm afraid we not do that, because curator need it https://cwiki.apache.org/confluence/display/CURATOR/TN13

Comment (yaooqinn) [reply kyuubi-metrics/pom.xml]: Thanks, @SteNicholas  I checked the guava usages in this module, I think we can remove guava here

**Review by SteNicholas — APPROVED** [review]

**Comment by SteNicholas** [comment]
LGTM.

**Comment by yaooqinn** [comment]
thanks @pan3793 @SteNicholas @ulysses-you for the help, merged to master

**Pull request closed without merging**